old-ui/app/config.js

```diff
@@ -249,6 +269,11 @@ function currentProviderDisplay (metamaskState) {
       value = 'Rinkeby Test Network'
       break
 
+    case 'classic':
+      title = 'Current Network'
+      value = 'Ethereum Classic Network'
+      break
```

Comment:
Similarly, you could add the ETC Testnet, [Morden](https://github.com/ethereumclassic/classic-guide/blob/master/source/network/test-networks.rst).